api/v1/controllers/HealthController.js

```diff
@@ -0,0 +1,19 @@
+var bodyParser = require('body-parser');
+var middleware = require('../middleware');
+
+var router = require('express').Router();
+
+function healthCheck(req, res, next) {
+    next();
+    return null;
+}
+
+router.use(bodyParser.json());
```

Comment:
Since you're not doing anything with the body, it seems like it'd be best not to even try to decode it. See if you can remove this line.